source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -388,18 +388,18 @@ FilterDataStatus Filter::onData(ProcessorState& state, Buffer::Instance& data, b
                ProcessorState::CallbackState::BufferedPartialBodyCallback) {
       // More data came in while we were waiting for a callback result. We need
       // to queue it and deliver it later in case the callback changes the data.
-      state.enqueueStreamingChunk(data, false, false);
+      state.enqueueStreamingChunk(data, end_stream, false);
```

Comment:
clean up the end_stream flag settings when the data is put in the queue. Same for a few such changes below.